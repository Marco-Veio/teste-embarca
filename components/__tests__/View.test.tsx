import React from "react";
import renderer from "react-test-renderer";

import { View, Text } from "../Themed";

describe("View component", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <View>
          <Text>Star Wars</Text>
        </View>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
