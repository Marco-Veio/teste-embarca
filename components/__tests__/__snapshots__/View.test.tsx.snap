// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View component renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
      },
      undefined,
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#000",
        },
        undefined,
      ]
    }
  >
    Star Wars
  </Text>
</View>
`;
